backend/cmm.mli

```diff
@@ -154,14 +154,12 @@ type trap_action =
   (** Remove the last handler from the trap stack. *)
 
 type trywith_kind =
```

Comment:
Why not just removing `trywith_kind`?

Reply:
It wasn't clear to me this was straightforward, but we should maybe try.

Reply:
I've done this now.